src/config.ts

```diff
@@ -76,6 +76,9 @@ const {
   STATSD_PORT,
   SYSTEM_ENVIRONMENT,
   UNLEASH_ADMIN_TOKEN,
+  UNLEASH_API,
+  UNLEASH_APP_NAME,
```

Comment:
@joeyAghion / @mc-jones - to confirm, this is just a raw string, correct? Its not something to configure in admin. Noticing we have `force-staging`, etc. I added here `metaphysics-staging`.

Reply:
Yea, I believe that is correct. Unleash should pick up the app name automatically, and expose some [details](https://unleash.artsy.net/applications/metaphysics-staging) about it in the UI. 

The more important config is the `UNLEASH_SERVER_KEY`.  I believe there needs to be two seperate keys. One set to `development` (staging and local) and one set to `production`. IIRC this is how Unleash allows you to toggle the feature on and off in different environments.

Reply:
Cool, i've updated this for staging and prod, via hokusai, so we should be good 👍